tests/manager/api/metadata/test_novelist.py

```diff
@@ -465,7 +466,8 @@ def test_get_items_from_query(self, novelist_fixture: NoveListFixture):
 
         # Set up a book for this library.
         edition = novelist_fixture.db.edition(
-            identifier_type=Identifier.ISBN, publication_date="2012-01-01"
+            identifier_type=Identifier.ISBN,
+            publication_date=dateutil.parser.parse("2012-01-01"),
```

Comment:
I'm not sure why this one changed.  I'm also not sure how it ever worked in the first place.